### PR TITLE
Add meeting minutes for Build WG Meeting 2016-04-26

### PR DESCRIPTION
**Pull request by Starefossen**
Signed-off-by: Hans Kristian Flaatten <hans.kristian.flaatten@dnt.no>

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2016-04-26.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+* Johan Bergström (@jbergstroem)
+* Michael Dawson (@mhdawson)
+* Myles Borins (@TheAlphaNerd)
+* Rich Trott (@Trott)
+
+## Standup
+
+* Alexis Campailla
+  * Running citgm on Windows. Figuring out how to add support in CI
+* Joao Reis (absent)
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-04-26.md]: nit: missing newline

**Comment by jbergstroem** [comment]
Nit otherwise LGTM. Thanks for doing this so quickly!

File: doc/meetings/build-wg-meeting-2016-04-26.md (hunk 1 above)
Comment (Starefossen) [inline doc/meetings/build-wg-meeting-2016-04-26.md]: Good catch 😄

**Comment by jbergstroem** [comment]
Btw, I don't think you need to do `git commit -s`; it seems very specific to the linux foundation:
http://developercertificate.org

**Comment by Starefossen** [comment]
Thanks for the heads up about `-s` :)

**Comment by mhdawson** [comment]
LBTM.

**Comment by orangemocha** [comment]
LGTM

**Comment by mhdawson** [comment]
Landed as 3795c65a9079de625a4949aa10290c1bf8613f49

**Pull request closed without merging**